serial_info_topic_name: /drivers/serial_info

gps_state_topic_name: /drivers/gps

protocol_name: GPCHC # GPGGA/GPCHC

record_filename: /home/nvidia/log/gps.csv

node_rate: 50   # [Herz]
